pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -295,7 +295,7 @@ func NewPriorityQueueWithClock(stop <-chan struct{}, clock util.Clock) *Priority
 	pq := &PriorityQueue{
 		clock:            clock,
 		stop:             stop,
-		podBackoff:       util.CreatePodBackoffWithClock(1*time.Second, 10*time.Second, clock),
+		podBackoff:       NewPodBackoffMap(1*time.Second, 10*time.Second),
```

Comment:
As you already have `NewPodBackoffMapWithDefaultTime()`, suggest to use that directly.

Reply:
Oh, I misread.. I thought all default backoff threshold is 10 seconds. Then don't bother. Keep them as is.